api/file.go

```diff
@@ -246,8 +249,21 @@ func handleImages(filenames []string, fileData [][]byte, teamId, channelId, user
 			// Create preview
 			go func() {
 				var preview image.Image
-				if width > int(utils.Cfg.FileSettings.PreviewWidth) {
-					preview = imaging.Resize(img, utils.Cfg.FileSettings.PreviewWidth, utils.Cfg.FileSettings.PreviewHeight, imaging.Lanczos)
+				ratio := float64(height) / float64(width)
```

Comment:
Why are we changing the size of the preview image? This will make the view image modal smaller for all new images.